apps/services/notifications/src/app/modules/notifications/connection.provider.ts

```diff
@@ -0,0 +1,72 @@
+import assert from 'assert'
+import {
+  SQSClient,
+  SQSClientConfig,
+  CreateQueueCommand,
+  GetQueueAttributesCommand,
+} from '@aws-sdk/client-sqs'
+import { SqsChannel } from '../../../types'
+
+const minute = 60
+const hour = minute * 60
+const day = hour * 24
+
+export const createQueue = async ({
+  mainQueueName,
+  deadLetterQueueName,
+  sqsConfig,
+}: {
+  mainQueueName: string
+  deadLetterQueueName: string
+  sqsConfig: SQSClientConfig
+}): Promise<SqsChannel> => {
+  const client = new SQSClient(sqsConfig)
+
+  // Create a dead letter queue
+  // https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-dead-letter-queues.html
+  const deadLetterQueue = await client.send(
+    new CreateQueueCommand({
+      QueueName: deadLetterQueueName,
+      Attributes: {
+        MessageRetentionPeriod: (day * 14).toString(),
```

Comment:
I agree that it might be a good idea to make these configurable.
If you however change the `Attributes` of an existing queue AWS will throw an error.
So to change the attributes of an existing queue you have to run another command.
I'd suggest we make it a separate task/ticket since this is not a simple as replacing these with environment constants.